internal/internal_versioning_client.go

```diff
@@ -113,6 +113,8 @@ type (
 		BuildID string
 		// Whether the worker is using the versioning feature.
 		UseVersioning bool
+		// An identifier to group task queues based on Build ID.
```

Comment:
This is very low-level description.  The point of a deployment is to _abstract_ the task queues.
I would say something like "A Deployment corresponds to a set of Worker instances you want deployed together.  For example, it may represent a Worker service.  Each time you deploy, you'll be changing the build ID for that deployment."